perception/traffic_light_map_based_detector/src/node.cpp

```diff
@@ -287,6 +287,11 @@ bool MapBasedDetector::getTrafficLightRoi(
   tier4_perception_msgs::msg::TrafficLightRoi & roi) const
 {
   roi.traffic_light_id = traffic_light.id();
+  if (pedestrian_tl_id_.find(traffic_light.id()) != pedestrian_tl_id_.end()) {
+    roi.traffic_light_type = tier4_perception_msgs::msg::TrafficLightRoi::PEDESTRIAN_TL;
+  } else {
+    roi.traffic_light_type = tier4_perception_msgs::msg::TrafficLightRoi::CAR_TL;
+  }
```

Comment:
```suggestion
  if (pedestrian_tl_id_.find(traffic_light.id()) != pedestrian_tl_id_.end()) {
    roi.traffic_light_type = tier4_perception_msgs::msg::TrafficLightRoi::PEDESTRIAN_TRAFFIC_LIGHT;
  } else {
    roi.traffic_light_type = tier4_perception_msgs::msg::TrafficLightRoi::CAR_TRAFFIC_LIGHT;
  }
```
Message type should not be shorten to avoid misunderstanding.

Reply:
fixed in
https://github.com/autowarefoundation/autoware.universe/commit/2f237e8a644b8b7ce191b74196f7a900e027bb80